pkgs/data/misc/mobile-broadband-provider-info/default.nix

```
@@ -9,6 +9,8 @@ stdenv.mkDerivation rec {
     sha256 = "1g9x2i4xjm2sagaha07n9psacbylrwfrmfqkp17gjwhpyi6w0zqd";
   };
 
+  nativeBuildInputs = [ (lib.getBin libxslt) ];
```

Just `libxslt` should be enough. And could you add a comment why it is needed?

```suggestion
  nativeBuildInputs = [
    # For xsltproc used for xyz.
    libxslt
  ];
```